docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -722,6 +722,79 @@ directly or used in conditional statements.
 * ``$file_size``
     Size of the file for this row.
 
+Functions
+---------
+
+The connector exposes the following table functions:
+
+system.table_changes
+^^^^^^^^^^^^^^^^^^^^
+
+Allows reading Change Data Feed (CDF) entries.
+This feature exposes row-level changes between two versions of a Delta table.
+When enabled on a Delta table, it records "change events" for all the data
+written into the table.
+In addition to columns present in the table this function
```

Comment:
How about something like

The function returns the following values for each change event:

and  then have a list with each field and a description

Reply:
I am not 100% this would be correct, I think that your version ommits the fact that values of table columns are also returned

Reply:
oh maybe I missunderstood I will try to apply it